xclim/indices/_agro.py

```diff
@@ -1261,16 +1449,38 @@ def standardized_precipitation_evapotranspiration_index(
     -----
     See Standardized Precipitation Index (SPI) for more details on usage.
```

Comment:
If results include NaNs, check that the offset parameter is larger than the minimum water budget values.